cvat-core/src/server-proxy.js

```diff
@@ -306,7 +306,8 @@
 
             async function exportDataset(id, format) {
                 const { backendAPI } = config;
-                let url = `${backendAPI}/tasks/${id}/dataset?format=${format}`;
+                const opBackendAPI = backendAPI.replace('api/v1', 'onepanelio/api/opv1');
```

Comment:
why not `onepanelio/api/v1` since the route is already under `onepanel`?

Reply:
Will change.